src/databricks/labs/ucx/install.py

```diff
@@ -235,8 +235,14 @@ def _prompt_for_new_installation(self) -> WorkspaceConfig:
         configure_groups.run()
         include_databases = self._select_databases()
         upload_dependencies = self.prompts.confirm(
-            f"Does given workspace {self.workspace_client.get_workspace_id()} " f"block Internet access?"
+            f"Does given workspace {self.workspace_client.get_workspace_id()} block Internet access?"
         )
+        wheelhouse: str | None = None
```

Comment:
hm... should not be here, per se..

Reply:
yes, this is something to be decided. As mentioned in [this comment](https://github.com/databrickslabs/ucx/pull/2380#pullrequestreview-2225047462), we can also ship the wheelhouse with the ucx install, then fetch it from there